src/components/App.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import Header from "./Header";
+import Footer from "./Footer";
+import Note from "./Note";
+import CreateArea from "./CreateArea";
+
+function App() {
+  // the useState hook for updating the list of notes on UI
+  // useState() takes in initial state as parameter
+  // returns an array containing the state and a setState function
+  const [list, setList] = useState([]);
+
+  // function addNote that adds a created note to list of notes
+  // takes in a note from the createArea component
+  const addNote = (note) => {
+    setList((prevValue) => {
+      return [...prevValue, note];
+    });
+  };
+
+  // function deleteNote deletes a note from list of notes
+  // takes in the note's id
+  const deleteNote = (id) => {
+    setList((prevValue) => {
+      return prevValue.filter((element, index) => {
+        return index !== id;
+      });
+    });
+  };
+
+  // function to create a single Note component
+  const createNote = (noteItem, index) => {
```

Comment:
The name is confusing. When I first read "createNote", I thought this function was creating a note. In fact, it doesn't. It returns the JSX for the note component. Maybe "noteComponent" or something similar would be more appropriate?